RecoLocalCalo/HGCalRecAlgos/interface/alpaka/HGCalRecHitCalibrationAlgorithms.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __HGCalRecHitCalibrationAlgorithms_H__
```

Comment:
```suggestion
#ifndef RecoLocalCalo_HGCalRecAlgos_interface_alpaka_HGCalRecHitCalibrationAlgorithms_h
```

Reply:
Done in https://github.com/cms-sw/cmssw/pull/45665/commits/f9e6cfac0101ac0d38179e820c185c91852f9618